ads/google/a4a/utils.js

```diff
@@ -172,6 +175,10 @@ export function googleAdUrl(
     if (isCanary(win)) {
       queryParams.push({name: 'isc', value: '1'});
     }
+    let eids = adElement.getAttribute('data-experiment-id');
```

Comment:
Rather than passing experimentIds, can't you just set them on data-experiment-id when you create the amp-ad element?